src/Clap-Core.package/ClapQueue.class/README.md

```diff
@@ -0,0 +1 @@
+I am a ClapMatcher subclass and I stock ClapCommand classes in a stack, so when matchOn: is invoqued, it invoque the matchOn: method of the ClapCommand which is at the top of the stack.If the matching is successful, it pops the ClapCommand, else if keeps the ClapCommand at the top
```

Comment:
invo**k**e
This is mostly paraphrasing the code. The goal here is to match positionals, since there can be other parameters in between them but should be recognized according to an overall sequence.